core/editor/src/document/document_message_handler.rs

```diff
@@ -652,13 +658,52 @@ impl MessageHandler<DocumentMessage, &InputPreprocessor> for DocumentMessageHand
 					}
 				}
 			}
-			FlipLayer(path, flip_horizontal, flip_vertical) => {
-				if let Ok(layer) = self.active_document_mut().document.layer_mut(&path) {
-					let scale = DVec2::new(if flip_horizontal { -1. } else { 1. }, if flip_vertical { -1. } else { 1. });
+			FlipSelectedLayers(axis) => {
+				// TODO: Handle folder nested transforms with the transforms API
+				let selected_paths = self.selected_layers_sorted();
+				if selected_paths.is_empty() {
+					return;
+				}
+
+				let selected_layers = selected_paths.iter().filter_map(|path| {
+					let layer = self.active_document().document.layer(path).unwrap();
```

Comment:
I don't know the answer, but is this a place you can use a `?` instead of `.unwrap()`?

Reply:
Yeah, good catch. We've generally been okay with using `unwrap` for layer paths provided to messages, but we might as well use `self.active_document().document.layer(path).ok()?`